great-idea-website/css/index.css

```diff
@@ -63,4 +63,141 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+
+.navigation-bar {
+	width: 850px;
+	top: 0;
+	border-bottom: 0px solid black;
+	position: fixed;
+	background-color: white;
+	width: 900px;
+	z-index: 5;
+}
+
+
+body {
+	margin: 0 auto;
+	width: 880px;
+	font-size: 16px;
+}
+
+header {
+	display: inline-block;
+	width: 100%;
+	position: relative;
+	background-color: white;
+	padding: 5px;
+}
+
+header nav a {
+	margin: 10px 28px;
+	font-size: 15px;
+	text-decoration: none;
+	color: grey;
+	display: inline-block;
+	background-color: white;
+}
+
+header nav a:first-of-type {
+	margin-left: 0;
+}
+
+header nav a:hover {
+	color: black;
+	font-size: 20px;
+	border-bottom: 2px dashed black;
+}
+
+
+.logo {
+	position: absolute;
+	right: 0;
+	padding-bottom: 4px;
+	background-color: white;
+}
+
+h1 {
+	font-size: 70px;
+}
+
+
+
+.first-section {
+	margin-top: 35px;
+	width: 880px;
+	height: 440px;
+	position: relative;
+	display: inline-block;
+	padding-top: 20px;
+}
+
+.first-section button {
+	border: 1px solid black;
+	background-color: white;
+	padding: 0 30px;
+	font-family: 'Titillium Web', sans-serif;
+	font-weight: 600;
+	font-size: 16px;
+	width: 170px;
+	height: 40px;
+	position: relative;
+}
+
+.left-side {
```

Comment:
I would like to see you be more specific with the code that you are grabbing here. This is most likely inside of a div or two and so you should select those as well. This is in case you decide later on that you want to call something else ".left-side" on another page, it won't break your code that way.